infra/src/test/scala/ch.epfl.scala.index.search/ESRepoTests.scala

```diff
@@ -19,7 +21,7 @@ class ESRepoTests extends AsyncFunSuite with Matchers with BeforeAndAfterAll {
 
   override protected def beforeAll(): Unit = {
     searchEngine.waitUntilReady()
-    searchEngine.reset()
+    Await.result(searchEngine.reset(), Duration.Inf)
```

Comment:
ahhhhhhhhhh! Is it the reason why the tests fail!

Reply:
:+1: